liquibase-core/src/main/java/liquibase/change/core/CreateViewChange.java

```diff
@@ -127,6 +133,7 @@ public void setEncoding(String encoding) {
         this.encoding = encoding;
     }
 
+    @DatabaseChangeProperty(description = "A brief descriptive comment")
```

Comment:
"A brief descriptive comment stored in the view metadata" (see similar comment on CreateTableChange).